src/index.js

```diff
@@ -21,6 +21,17 @@ function create(operator, value) {
         $options: 'i'
       }
     };
+  } else if (operator === 'is any of') {
+    if (!Array.isArray(value)) throw new Error(`When called with argument 'operator' 'is any of', expected argument 'value' to be of type 'Array'.`);
+    const conditions = value.map((condition) => {
+      return ({
```

Comment:
Ah, there is none. I just came over from React-land where parens around multi-line returns are necessary 🙃